store/src/java-test/com/zimbra/cs/mailbox/calendar/ZAttendeeTest.java

```diff
@@ -16,16 +16,44 @@
  */
 package com.zimbra.cs.mailbox.calendar;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Iterator;
+import java.util.List;
+
 import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
 import com.zimbra.common.calendar.ZCalendar.ICalTok;
+import com.zimbra.common.calendar.ZCalendar.ZComponent;
 import com.zimbra.common.calendar.ZCalendar.ZParameter;
 import com.zimbra.common.calendar.ZCalendar.ZProperty;
 import com.zimbra.common.service.ServiceException;
+import com.zimbra.common.util.ByteUtil;
+import com.zimbra.cs.account.Provisioning;
+import com.zimbra.cs.mailbox.MailboxTestUtil;
+import com.zimbra.cs.mime.ParsedMessage;
+import com.zimbra.cs.mime.ParsedMessage.CalendarPartInfo;
 
 public class ZAttendeeTest {
 
```

Comment:
Can you add the following, so we can find which test fails in Circle CI
 @Rule public TestName testName = new TestName();
    @Rule public MethodRule watchman = new ZTestWatchman();

Reply:
Fixed.